wmi.go

```diff
@@ -226,11 +236,14 @@ func (c *Client) Query(query string, dst interface{}, connectServerArgs ...inter
 		return ErrInvalidEntityType
 	}
 
-	lock.Lock()
-	defer lock.Unlock()
+	c.lock.Lock()
```

Comment:
Lock to be reviewed later, most likely protecting sbwem service/connect server stuff to be accessed concurrently